CapybaraLearning/spec/menu_spec.rb

```diff
@@ -0,0 +1,185 @@
+require_relative '../all_requires.rb'
+
+describe 'When testing demo.opensourcecms website' do
+
+  before(:all) do
+    Capybara.configure do |config|
+      config.run_server = false
+      config.default_driver = :poltergeist
+    end
+      @login_page = LoginPage.new
+      @home_page = HomePage.new
+      @updates_page = UpdatesPage.new
+      @all_posts_page = AllPostsPage.new
+      @categories_page = CategoriesPage.new
+      @tags_page = TagsPage.new
+      @all_pages_page = AllPagesPage.new
+      @comments_page = CommentsPage.new
+      @appearance_page = AppearancePage.new
+      @customize_page = CustomizePage.new
+      @widgets_page = WidgetsPage.new
+      @menus_page = MenusPage.new
+      @header_page = HeaderPage.new
+      @plugins_page = PluginsPage.new
+      @user_page = UsersPage.new
+      @profile_page = YourProfilePage.new
+      @tools_page = ToolsPage.new
+      @import_page = ImportPage.new
+      @export_page = ExportPage.new
+      @site_health_page = SiteHealthPage.new
+      @settings_general_page = SettingsGeneralPage.new
+      @writing_page = WritingPage.new
+      @reading_page = ReadingPage.new
+      @discussion_page = DiscussionPage.new
+      @permalink_page = PermalinkPage.new
+      @privacy_page = PrivacyPage.new
+  end
+
+  it 'Verify navigating to the login page' do
+    @login_page.load
+  end
+
+  it 'Verify loggining' do
+    @home_page = @login_page.log_in('opensourcecms', 'opensourcecms')
+    expect(@home_page.loggined?).to be true
+  end
+
+  it 'Verify navigating to Home page' do
```

Comment:
Try to write common scenario for verifying navigation to all pages to reduce the code. The approach might be the following: create array of pages and perform common scenario for each of them. You might need to rework approach with messages on each page class, for example, create 'on_page?' method.